bait-files/src/main/webapp/userHistory.js

```diff
@@ -0,0 +1,52 @@
+const url =
+    'https://step-bait-project-2020.uc.r.appspot.com/review.html?reviewId=';
+
+function getCompletedReviews() {
+  fetch('/userHistory?is-pending-review=false')
+      .then(response => response.json())
+      .then((reviews) => {
+        for (review of reviews) {
+          addReview(review, document.getElementById('completed-table'));
+        }
+      });
+}
+
+function getPendingReviews() {
+  fetch('/userHistory?is-pending-review=true')
+      .then(response => response.json())
+      .then((reviews) => {
```

Comment:
Same comment about being consistent with whether or not you want to use parentheses for a single parameter of an arrow function.

Reply:
Removed